lib/playgroundbook_renderer/chapter_collator.rb

```diff
@@ -11,47 +11,29 @@ def initialize(page_writer = PageWriter.new, ui = Cork::Board.new)
       @ui = ui
     end
 
-    def collate!(chapter_name, chapter_file_contents, imports)
+    def collate!(chapter_name, parsed_chapter, imports)
```

Comment:
nah, I wouldn't call it that then 👍 -  the gem owns all that sandbox ( as they're runtime generated ) think it's safe to not call it mutating